client/jetpack-cloud/sections/partner-portal/license-lightbox/single-site-upsell-lightbox.tsx

```diff
@@ -59,29 +54,17 @@ export default function SingleSellUpsellLightbox( {
 			} )
 		);
 		onHideLicenseInfo();
-		page(
-			addQueryArgs(
-				{
-					product_slug: currentProduct.slug,
-					source: 'dashboard',
-					site_id: siteId,
-				},
-				'/partner-portal/issue-license/'
-			)
-		);
-	}, [ currentProduct, dispatch, onHideLicenseInfo, siteId ] );
-
-	const onProceedToCheckout = useCallback( () => {
-		dispatch(
-			recordTracksEvent( 'calypso_jetpack_single_site_upsell_proceed_to_checkout_click', {
-				product: currentProduct.slug,
-			} )
-		);
-		onHideLicenseInfo();
-	}, [ currentProduct, dispatch, onHideLicenseInfo ] );
+		submitForm( [
+			{
+				...currentProduct,
```

Comment:
This didn't add `siteId` to the URL until I added this:

```suggestion
				...currentProduct,
				siteId,
```

Reply:
Ah, thank you!

Reply:
Ignore this; probably a Calypso artifact.